entrypoint.sh

```diff
@@ -41,5 +45,22 @@ if ! nexus-allowlist --admin-password "$NEXUS_ADMIN_PASSWORD" --nexus-host "$NEX
     exit 1
 fi
 
-# Run allowlist configuration now, and again whenever allowlist files are modified
-find "$ALLOWLIST_DIR"/*.allowlist | entr -n nexus-allowlist --admin-password "$NEXUS_ADMIN_PASSWORD" --nexus-host "$NEXUS_HOST" --nexus-port "$NEXUS_PORT" update-allowlists --packages "$NEXUS_PACKAGES" --pypi-package-file "$PYPI_ALLOWLIST" --cran-package-file "$CRAN_ALLOWLIST"
+if [ -n "$ENTR_FALLBACK" ]; then
+    echo "$(timestamp) Using fallback file monitoring"
+    # Run allowlist configuration now
+    nexus-allowlist --admin-password "$NEXUS_ADMIN_PASSWORD" --nexus-host "$NEXUS_HOST" --nexus-port "$NEXUS_PORT" update-allowlists --packages "$NEXUS_PACKAGES" --pypi-package-file "$PYPI_ALLOWLIST" --cran-package-file "$CRAN_ALLOWLIST"
+    # Periodically check for modification of allowlist files and run configuration again when they are
+    hash=$(hashes)
+    while true; do
+        new_hash=$(hashes)
+        if [ "$hash" != "$new_hash" ]; then
```

Comment:
Not sure how well `hashes` will compare here as the string will include new lines and look something like:

```
1504b4d2daae6afcaf5d9a396e580ba2  /allowlists/cran.allowlist
fdf8764e4d5f0583abcb5753df8204f2  /allowlists/pypi.allowlist
```

Have you tested that this works?

Reply:
Yep :+1:, it works for either file.
I imagine it is just comparing strings which happen to have `\n` in them.